test/specs/request-parser.spec.js

```diff
@@ -676,7 +676,7 @@ for (let spec of specs) {
             expect(req.body).to.deep.equal({
               foo: "bar",
               biz: ["42", "43", "44"],
-              baz: ["B", "C", "A"],
+              baz: ["B", undefined, "C", undefined, undefined, "A"],
```

Comment:
This is a weird change.  Is this because of the Multer upgrade?
I'm fine with it, but definitely need to call it out in the changelog for 4.0.0.

Reply:
Yea I thought I had left a comment for it. If you look at the params it basically puts them exactly to the index in the array they are assigned and fills in the blanks with `undefined`. I will note in changelog.

Reply:
Looking around more I see form-url-encoded would be ["B", "C", "A"] - I don't know how well I could write up something that would make this compatible but it's odd that the two different content-types would express this slightly differently. Thoughts on what to do there?

Reply:
Yeah, it's weird.  But I think this is an edge-case.  We'll just consider it the new normal and expected behavior, starting with 4.0.0